src/pages/Challenge.js

```diff
@@ -76,7 +78,7 @@ const Challenge = ({ classes }) => {
 		<>
 			<div className={classes.background} />
```

Comment:
You shouldn't need this background div if you aren't using the png anymore. The background color can just be applied to some `app` level wrapper component, or even in `theme.css` as the `backgroundColor`

Reply:
When you get rid of this, make sure you delete the styles above as well